preprocessy/outliers/_handleoutlier.py

```diff
@@ -47,53 +44,59 @@ def __init__(self, train_df, params):
         self.quartiles = {}
         self.first_quartile = 0.05
         self.third_quartile = 0.95
-        if 'cols' in params.keys():
-            self.cols = params['cols']
-        if 'removeoutliers' in params.keys():
-            self.remove_outliers = params['removeoutliers']
-        if 'replace' in params.keys():
-            self.replace = params['replace'] 
-        if 'q1' in params.keys():
-            self.first_quartile = params['q1']
-        if 'q3' in params.keys():
-            self.third_quartile = params['q3']
-
-    def __return_quartiles(self,col):
+        if "cols" in params.keys():
+            self.cols = params["cols"]
+        if "removeoutliers" in params.keys():
+            self.remove_outliers = params["removeoutliers"]
+        if "replace" in params.keys():
+            self.replace = params["replace"]
+        if "q1" in params.keys():
+            self.first_quartile = params["q1"]
+        if "q3" in params.keys():
+            self.third_quartile = params["q3"]
+
+    def __repr__(self):
+        params_default = dict(([], {}))
```

Comment:
Rather than making an empty dictionary, use the values of the corresponding attributes of `self` if present else `None`.